iroh-net/src/magicsock.rs

```diff
@@ -2934,7 +2930,7 @@ pub(crate) mod tests {
             };
         }
 
-        for i in 0..10 {
+        for i in 0..100 {
```

Comment:
can we bring this back down to 10 when we want to merge?

Reply:
yeah, I just wanted to make sure this is stable first